### PR TITLE
Add error view component to app & app states

### PR DESCRIPTION
**Pull request by FaizChishtie**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -58,6 +69,20 @@ export const App = (props: IAppProps) => {
         <SettingsView />
       </div>
     );
+  } else if (AppState === AppStateEnum.Loading) {
+    return (
```
Comment (sachingupta) [inline src/App.tsx]: we should move this into separate component(ContentView) similar to SettingsView

File: src/App.tsx [hunk 2]
```diff
@@ -58,6 +69,20 @@ export const App = (props: IAppProps) => {
...
+      <div>
+        <SearchBar onSearch={handleSearch} onViewChange={handleViewChange} />
```
Comment (sachingupta) [inline src/App.tsx]: Looks like SearchBar we are rending always. lets keep it outside of if else

File: src/css/App.css [hunk 3]
```diff
@@ -76,4 +76,11 @@
   border-radius: 3px 3px 0px 0px;
   margin: 2px 2px 0 0;
   box-shadow: 0 .2rem .4rem -.075rem rgba(0, 0, 0, .1);
+}
+
+.loadIcon {
```
Comment (sachingupta) [inline src/css/App.css]: we can move this to its own css file loadIcon.css

File: src/components/LoadIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Loader } from '@stardust-ui/react';
+
+export const LoadIcon = (props: any) => {
+  return <div className="loadIcon"> {props.isLoading ? <Loader /> : null} </div>;
```
[the remark this answers is not in the document]
Comment (FaizChishtie) [reply src/components/LoadIcon.tsx]: yeah we merged

File: src/App.tsx (hunk 2 above)
Comment (FaizChishtie) [reply src/App.tsx]: sure

**Pull request merged**